doc/installing.md

```diff
@@ -64,6 +64,20 @@ Complete the following steps to install the snap:
 
         sudo snap install lxd --channel=5.0/stable
 
+```{note}
+LXD cluster members all need to use the exact same version of LXD and this requires special consideration due to how snaps are deployed as explained in {ref}`Keep cluster members in sync<howto-snap-cohort>`. To ensure all the cluster members are offered the same snap revision, it is possible to specify a cohort parameter which will bypass the progressive deployment:
```

Comment:
@minaelee @simondeziel I wonder whether this should be in https://documentation.ubuntu.com/lxd/en/latest/howto/cluster_manage/#upgrade-cluster-members and linked to from here or vice versa. There's significant overlap in these pages.

Reply:
I do agree that our installation/upgrade/snap doc is all over the place and especially messy when we consider clusters needing cohorts often wanting holds.

That doc reorg feels like a bigger task, a needed one that's for sure. Do you have any objections to having it split off from that PR though?